doc/content/source/nodal_kernels/TrappingNodalKernel.md

```diff
@@ -1,20 +1,53 @@
 # TrappingNodalKernel
 
-!alert construction title=Undocumented Class
-The TrappingNodalKernel has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /NodalKernels/TrappingNodalKernel
 
 ## Overview
 
-!! Replace these lines with information regarding the TrappingNodalKernel object.
-
-## Example Input File Syntax
-
-!! Describe and include an example of how to use the TrappingNodalKernel object.
+This object implements the residual
+
+\begin{equation}
+\frac{\alpha_t}{N} C_t^e C_s
+\end{equation}
+
+where $\alpha_t$ is the trapping rate coefficient, which has dimensions of
+1/time, $N$ is the atomic number density of the host material, $C_t^e$ is the
+concentration of empty trapping sites, and $C_s$ is the concentration of the
+mobile species.
+
+As outlined in [content/index.md#scaling] it is important to scale different
```

Comment:
This is breaking. Try just `[index.md#scaling]` since you've already included the content dir in your config.yml file.